lib/mpl_toolkits/axes_grid1/axes_rgb.py

```diff
@@ -55,6 +56,7 @@ def make_rgb_axes(ax, pad=0.01, axes_class=None, add_all=True):
     return ax_rgb
 
 
+@cbook.deprecated("3.3", alternative="ax.imshow(np.dstack([r, g, b]))")
```

Comment:
Can we re-implement the function that way, too? It's currently pretty inefficient.

Reply:
done